README.rst

```diff
@@ -39,4 +39,90 @@ applications on top of SQLAlchemy.
 
 AnyBlok is released under the terms of the `Mozilla Public License`.
 
-See the `latest documentation <http://doc.anyblok.org/>`_
+AnyBlok is hosted on `github <https://github.com>`_ - the main project
+page is at https://github.com/anyblok/anyblok or
+http://code.anyblok.org. source code is tracked here
+using `git <https://git-scm.com>`_.
+
+Releases and project status are available on Pypi at
+https://pypi.python.org/pypi/anyblok.
+
+The most recent published version of the documentation should be at
+https://doc.anyblok.org.
+
+There is a tutorial to teach you how to develop applications with AnyBlok at
+https://anyblok.gitbooks.io/anyblok-book/content/en/
+
+Project Status
+--------------
+
+AnyBlok is currently in alpha status and is expected to be fairly
+stable.
+
+Users should take care to report bugs and missing features on an as-needed
+basis.
+
+It should be expected that the development version may be required
+for proper implementation of recently repaired issues in between releases;
+the latest master is always available at https://github.com/AnyBlok/AnyBlok/archive/master.zip.
+
+Installation
+------------
+
+Install released versions of AnyBlok from the Python package index with
+`pip <http://pypi.python.org/pypi/pip>`_ or a similar tool::
+
+    pip install anyblok
+
+Installation via source distribution is via the ``setup.py`` script::
+
+    python setup.py install
+
+Installation will add the ``anyblok`` commands to the environment.
+
+Running Tests
+-------------
+
+To run framework tests with ``pytest``::
+
+    pip install pytest
+    ANYBLOK_DATABASE_DRIVER=postgresql ANYBLOK_DATABASE_NAME=test_anyblok py.test anyblok/tests
+
+To run tests of all installed bloks::
+
+    anyblok_createdb --db-name test_anyblok --db-driver-name postgresql --install-all-bloks
+    ANYBLOK_DATABASE_DRIVER=postgresql ANYBLOK_DATABASE_NAME=test_anyblok py.test anyblok/bloks
+
+AnyBlok is tested continuously using `Travis CI
+<https://travis-ci.org/AnyBlok/AnyBlok>`_
+
+Contributing (hackers needed!)
+------------------------------
+
+AnyBlok is at a very early stage, feel free to fork, talk with core dev, and spread the word!
```

Comment:
@jssuzanne how early stage is it ? we may afraid developer that this project is not ready for production usage